src/foam/u2/view/LoginView.js

```diff
@@ -291,19 +286,19 @@ foam.CLASS({
         .end()
       // deciding to render half screen with img and model or just centered model
         .callIfElse( !! this.imgPath && !! split, () => {
-          this.add(split)
-          .start()
+          split.leftPanel.add(this.E().start()
```

Comment:
Why leftPanel.add(this.E().start()...
instead of just:
leftPanel.start()...
?

Reply:
Fixed